Resources/doc/reference/configuration/headers.rst

```diff
@@ -229,6 +229,22 @@ This enables a simplistic ETag calculated as md5 hash of the response body:
                     headers:
                         etag: true
 
+
+.. versionadded:: 2.2
+
+You can set up ETag to be strong or weak by passing values "strong", "weak".
+Value true is still treated as 'strong' due to backward compatibility.
```

Comment:
lets remove this line here, this should only be in the code and the CHANGELOG, but not in the doc.